### ISSUE TITLE
Respect the language choice

### ISSUE THREAD

**Issue by Yoshi2889** [Antergos/Cnchi#729]
On the first page, you choose a language. On the second page you choose a region. Cnchi seems to prefer the latter when setting a language on your system. Some users might want to pick English when not living in a country where English is spoken natively, for example. 

To reproduce :
- Pick English or a different language than natural for a country. 
- pick a country where the selected language is not the main language. 
- Install. 
- Find that the graphical language is the country's native language after all.

**Comment by karasu** [comment]
Yes... that's an old bug that we haven't got time to pinpoint. Just a question, did you install the Gnome DE?

**Comment by Yoshi2889** [comment]
I picked Xfce this time.

**Comment by karasu** [comment]
Ups, then it's not the bug I was thinking of.

Can you detail your example?
Which language, country and territory did you pick up? Which language did you end with in XFCE?

**Comment by Yoshi2889** [comment]
I picked English for language, Netherlands (NL) for territory, and ended up with Dutch in Xfce.

**Comment by dacki** [comment]
Happens also with Gnome DE.
Language/Locale: en_US, Location: DE, Keyboard: DE
Result: Locale All German but the Keyboard layout EN

**Comment by karasu** [comment]
First screen lets you choose the installer language, not the installed system language.
The system's installed language (thus its locale) is determined by the location screen. See here:

![locale_info](https://user-images.githubusercontent.com/175797/34467213-03daea32-eeeb-11e7-9bc4-dc9d865996d5.png)

As the text warns, the location (territory) is used to choose your locale. Most of the time it will be where you reside, but if you want to use another locale you must choose the region where that locale is from.

I know it's a bit confusing, to say the least.

I'll try to clarify it a bit more in the installer text warnings.

@dacki yours is another issue. The keyboard layout is chosen in another page. Problem is that in your case keyboard layout was not set correctly in Gnome.

**Comment by karasu** [comment]
@Yoshi2889 
>I picked English for language, Netherlands (NL) for territory, and ended up with Dutch in Xfce.

Cnchi chooses Dutch because you tell it that your location is Netherlands. If you want to use English you should choose another location (United States, for instance). And then choose NL timezone, of course.

@ZeWebDev 

>I picked English language and locale, but i chose Portugal as the location, and my installation was mostly in Portuguese.

Same here. In your case, choosing Portugal for your locale means that you will be using Portuguese. Again, if you want to use English you must choose an English location.

Confusion here lays on that the location you choose is not always were you reside (but it is most of the time). The location is used to choose locale (thus, language). If you want to use a different language from the location you reside you must choose another location. Maybe the "location" word is wrongly used here... but I lack a best word for it.

This confusion does not happen to people that want their system in the same language that is spoken in their location.

There's another problem, and that is that you chose another location you'll end up with that location locale settings (not only language, but everything else, like date format for instance). To change this you should setup your locale settings manually in your DE settings manager.

See this: https://wiki.archlinux.org/index.php/locale and this: https://wiki.gentoo.org/wiki/Localization/Guide

We're open for suggestions/criticisms ...

**Comment by Yoshi2889** [comment]
@karasu however that doesn't really make sense, it should be noted that the language you choose is just for the installer and not for your actual installation.
